content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -74,8 +74,11 @@ Kubernetes sometimes checks authorization for additional permissions using speci
 of the `bind` verb on `roles` and `clusterroles` resources in the `rbac.authorization.k8s.io` API group.
 * [Authentication](/docs/reference/access-authn-authz/authentication/) layer checks for authorization of the `impersonate` verb on `users`, `groups`, and `serviceaccounts` in the core API group, and the `userextras` in the `authentication.k8s.io` API group.
 
-## Authorization Modules
- * **Node** - A special-purpose authorizer that grants permissions to kubelets based on the pods they are scheduled to run. To learn more about using the Node authorization mode, see [Node Authorization](/docs/reference/access-authn-authz/node/).
+## Authorization Modes {#authorization-modules}
+
```

Comment:
hello @jaypipes . Should the link stay as `modules`?

Reply:
Hi @kbhawkey, yeah, I left the section hyperlink as "modules" in order to prevent breaking any existing links to this section, but changed the section title element to correctly say "modes".